boards/microchip-pic32mx-generic/README.md

```diff
@@ -123,6 +123,18 @@ Caveats:
   running at the appropriate frequency. This is a problem due to the CAN clock
```

Comment:
Could we make sure that this mark down file gets picked up by Doxygen?

Reply:
done